packages/gafl-webapp-service/src/pages/concessions/date-of-birth/date-of-birth.njk

```diff
@@ -3,6 +3,7 @@
 {% from "date-input/macro.njk" import govukDateInput %}
 {% from "inset-text/macro.njk" import govukInsetText %}
 {% set title = mssgs.dob_title_you if data.isLicenceForYou else mssgs.dob_title_other %}
+{% set describedByTag = "date-of-birth-hint" %}
```

Comment:
wondering if way to do this in route so can test?

Reply:
As I see this, it's the id for an element on the page which is passed to a sub-template. As such, I think we can get away with just having it in the template rather than needing to have it in the getData function. However, I do think that if we're using a variable in this way, the div with this id should use the variable rather than having the id hard-coded.